### ISSUE TITLE
Warnings/error when finetune UCF101 

### ISSUE THREAD

**Issue by murilovarges** [facebookresearch/R2Plus1D#12]
Hi,

I'm trying to reproduce UCF101 results, but when I try to fine tune in UCF101 dataset some messages are displayed and the accuracy is very low.

**Displayed Messages:**
_1. Invalid return value 0 for stream protocol
2. [NULL @ 0x7dcac81069c0] Failed to parse extradata
3. [mpeg4 @ 0x7dcacc0a3600] Video uses a non-standard and wasteful way to store B-frames ('packed B-frames'). Consider using the mpeg4_unpack_bframes bitstream filter without encoding but stream copy to fix it._

**File output:**
[https://pastebin.com/rqhZtxSD](https://pastebin.com/rqhZtxSD)

**Comment by murilovarges** [comment]
I notice when I run the test script  _"/pytorch/caffe2/python/operator_test/ video_input_op_test.py"_ with some videos from UCF101 same messages are displayed is occurring but test ran with success.

**Comment by dutran** [comment]
If the training/test loss look fine, you can safely ignore the decoding warnings.
